grapher_admin/migrations/0005_datasetcommithistory.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2017-07-11 03:26
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('grapher_admin', '0004_datasetcategory_fetcher_autocreated'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='DatasetCommitHistory',
+            fields=[
```

Comment:
Yep, this can be done using git log. Although it might be slower to query for commit info from git itself instead of looking up from the db, it is a better design to have commit info extracted from git rather than having duplicate info in two places.